src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -631,12 +632,13 @@ def forward(
                 `input_features`, the [`WhisperFeatureExtractor`] should be used for extracting the mel features,
                 padding and conversion into a tensor of type `torch.FloatTensor`. See
                 [`~WhisperFeatureExtractor.__call__`]
+            attention_mask (`torch.Tensor`)`, *optional*):
+                Whisper does not support masking of the `input_features`, this argument is preserved for compatibility, but it is not used. By default the silence in the input log mel spectrogram are ignored. 
```

Comment:
`make style` will fix this veeeeeery long line ;-)

Reply:
It was a good suggestion I accepted from GH suggestion from @ArthurZucker .

:)